tests/create_task_resources.yml

```diff
@@ -0,0 +1,49 @@
+name: Create Tes Task Job With Resources
+description: Job to create a new TES Task with resources
+service: TES
+compatible_versions:
+  - v1.0
+  - v1.1
+tags:
+  - logical
+  - create task
+  - create task test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "CompTest",
+        "description": "CompTest",
+        "executors": [
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString1",
+              "TestString2"
+            ]
+          },
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString3",
+              "TestString4"
+            ]
+          }
+        ],
+        "resources": {
+          "cpu_cores": 4,
+          "preemptible": false,
+          "ram_gb": 8,
+          "disk_gb": 40,
+          "zones": [
```

Comment:
This test is also not related to the new `v1.1` features, right?  As for a general schema check, `resources` and its children are optional, so adding a test wont be beneficial.

Reply:
Good point! If all properties (including nested ones) are optional _and_ `additionalProperties` is not set to `false`, then tests like these don't really add value.

And I think that's indeed the case here, and possibly elsewhere.